components/SuccessSection/index.js

```diff
@@ -0,0 +1,56 @@
+import { useRouter } from "next/router";
+import { StyledButton } from "../../styles/StyledButton";
+import { StyledRoundCounter } from "../../styles/StyledRoundCounter";
+import { useEffect, useState } from "react";
+import useAudio from "../useAudio";
+import useStore from "../useStore";
+import { StyledInstruction } from "../../styles/StyledInstruction";
+import { StyledQuote } from "../../styles/StyledQuote";
+
+export default function SuccessSection({ breathIntervalDelay }) {
+  const resetIceBathCountdown = useStore(
+    (state) => state.resetIceBathCountdown
+  );
+  const switchIceBathCountdownIsActive = useStore(
+    (state) => state.switchIceBathCountdownIsActive
+  );
+  const iceBathCountdownIntervalId = useStore(
+    (state) => state.iceBathCountdownIntervalId
+  );
+  const resetIceBathCount = useStore((state) => state.resetIceBathCount);
+  const switchIceBathCountIsActive = useStore(
+    (state) => state.switchIceBathCountIsActive
+  );
+  const iceBathIntervalId = useStore((state) => state.iceBathIntervalId);
+
+  const router = useRouter();
+
+  const { playGong } = useAudio({ breathIntervalDelay });
+  const [isGongActive, setIsGongActive] = useState(true);
+
+  useEffect(() => {
```

Comment:
Maybe i am not understanding things... But can isGongActive ever be anything but `true`?

If not, then maybe we only need 1 useEffect. 

``` js 
useEffect(() => {
  playGong();
}, []);
```

We could also get rid of the useState then 

`const [isGongActive, setIsGongActive] = useState(true);`